knoten/bundle.py

```diff
@@ -373,3 +406,107 @@ def compute_residuals(network, sensors):
 
     V = V.reshape(num_meas*2)
     return V
+
+def update_parameters(sensors, parameters, network, updates, coefficient_columns):
+    """
+    Updates the sensor objects parameter values and the ground point values in the
+    networks DataFrame. The update occurs directly to variables, so nothing is returned.
+
+    Parameters
+    ----------
+    sensors  :  dict
+                A dictionary that maps ISIS serial numbers to CSM sensors
+    parameters :  list
+                  The list of  CsmParameter to compute the partials W.R.T.
+    network  :  DataFrame
+                The control network as a dataframe generated by plio.
+    updates   : np.ndarray
+                An array of updated parameter values
+    coefficient_columns:  OrderedDict
+                          Dictionary that maps serial numbers and point IDs to
+                          the column range their parameters are in the Jacobian
+                          matrix.
+
+    Returns
+    -------
+
+    """
+    # update the sensor partials
+    for sn, sensor in sensors.items():
+        for i, param in enumerate(parameters[sn]):
+            if i > coefficient_columns[sn][1]:
+                print('THIS SHOULD BE AN ACTUAL ERROR')
```

Comment:
This this PR a good time to update this to an actual error, or should this get changed in the future?